test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_but_respond.lua

```diff
@@ -0,0 +1,101 @@
+--  Requirement summary:
+--  [HeartBeat][Genivi]: SDL must track sending of HeartBeat_request from/to mobile app
+--
+--  Description:
+--  Check that no heartbeat occurs if App uses v3 protocol version and doesn't send HB to SDL, 
```

Comment:
@LitvinenkoIra, may be will be better if you write heartbeat timeout, instead only heartbeat.